core/camel-core/src/test/java/org/apache/camel/component/file/FileConsumerFailureHandledTest.java

```diff
@@ -57,7 +57,7 @@ public void testLondon() throws Exception {
         mock.expectedBodiesReceived("London");
 
         template.sendBodyAndHeader(fileUri(), "London", Exchange.FILE_NAME, "london.txt");
-        mock.assertIsSatisfied(1000);
+        mock.assertIsSatisfied(3000);
```

Comment:
Why the timeout increase?

Reply:
I couldn't find another explanation why this test has failed on a previous PR check run, as locally it passes when I run all camel-core tests. The same I see now but with other tests. After this change of timeout, this `FileConsumerFailureHandledTest` passes but two others fail: 
- `org.apache.camel.processor.MulticastParallelTimeoutStreamCachingTest.testCreateOutputStreamCacheBeforeTimeoutButWriteToOutputStreamCacheAfterTimeout`
- `org.apache.camel.processor.MulticastParallelStreamingTest.testMulticastParallel`

Both don't use temp directory anyhow and shouldn't be affected by my changes, so I'm wondering how stable such tests are. I'm very new to it so I'm trying to understand if it's something that can happen ocassionally due to slower test execution, or if it's related to my PR changes. I apologize for my confusion as someone new, but really appreciate your guidance and open to suggestions 🙂

Reply:
So, the challenge here is that we have to balance two things: 

1. Quick test execution 
2. Stability 


In some of the systems where these tests run, such as GitHub, the systems running them could be overloaded at the time of execution. Some tests use fixed sleeps and/or timeouts to wait for "something" to happen, but this process is not stable at all (hence, why we have been converting most of the sleeps with Awaitility). 

I suspect that's why you see these new failures: maybe if you run them again, you'll see a different set of failures. 

Normally, what we do is that we use the [ASF CI](https://ci-builds.apache.org/job/Camel/job/Camel%20JDK17/) as the baseline: as long as tests run fine there, we are OK with it (and we are a bit lenient with [most of] the core check failures on GitHub).

So, in this case, I'd leave it as is.

Reply:
Thanks for the explanation, I will revert the timeouts then and set them back to 1000

Reply:
Reverted back to 1000